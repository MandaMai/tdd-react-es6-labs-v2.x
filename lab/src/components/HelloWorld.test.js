import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

import HelloWorld from './HelloWorld';

it('matches the snapshot', () => {
    const component = shallow(<HelloWorld />);
    expect(toJson(component)).toMatchSnapshot();
});