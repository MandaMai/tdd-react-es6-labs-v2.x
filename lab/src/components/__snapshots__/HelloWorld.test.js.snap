// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<h1>
  Hello, 
  !
</h1>
`;
